reactioncommerce/reaction
# Fix #3821: use numericInput for pricing

Resolves #3821  
Impact: **major**  
Type: **bugfix**

## Issue
The component's value was changing because stale value was being stored in state.

## Solution
Used the `NumericInput` component modified by MJ for pricing.

## Testing
- As an admin, set the shop base currency to EUR.
- Add a product
- Input a value for price.
- The value **will not** get converted as soon as the field blurs.
- Publish the product, and observe that the price value was retained as is.
